rust/connlib/snownet/src/node.rs

```diff
@@ -766,21 +793,8 @@ where
         allowed_stun_servers: &HashSet<SocketAddr>,
         allowed_turn_servers: &HashSet<SocketAddr>,
     ) {
-        for local in self.local_interfaces.iter().copied() {
```

Comment:
I'm a bit confused as to why this wouldn't have gathered the host candidate used to connect to the relay as well. 

Am I understanding the problem correctly?:

- local interfaces are gathered
- A new socket is opened to the Relay (using source address `local` which is `0.0.0.0`?)
- We gather the resulting socket addrs (`0.0.0.0:8452`)

These are all useless because we can't see the actual source socketaddr the OS picked for us?

Reply:
Yep, `0.0.0.0` doesn't pick an address for you, it means all interfaces.